PostProcessing/Editor/PostProcessLayerEditor.cs

```diff
@@ -162,6 +162,9 @@ void DoAntialiasing()
                     if (RuntimeUtilities.isSinglePassStereoSelected)
                         EditorGUILayout.HelpBox("TAA requires Unity 2017.3+ for Single-pass stereo rendering support.", MessageType.Warning);
                     #endif
+                    var camera = m_Target.GetComponent<Camera>();
```

Comment:
`GetComponent<>()` generates a lot of garbage, I know it's in the editor but as the inspector is redrawn multiple time per second... Let's not put more pressure on it :) Please cache the camera in the `OnEnable`.